ui.frontend/src/main/webpack/site/js/buildSearch.ts

```diff
@@ -20,7 +20,27 @@ export const buildSearch = async (
     return
   }
 
-  const { callbacks, autoSuggestionDebounceTime = 500 } = options || {}
+  const { trackingUrl } = searchConfig
+
+  const { callbacks = {}, autoSuggestionDebounceTime = 500 } = options || {}
+
+  if (trackingUrl) {
+    callbacks.onSearchItemClick = (url: string) => {
+      const headers = new Headers()
+      headers.append('Content-Type', 'application/x-www-form-urlencoded')
+      fetch(`${window.location.origin}${trackingUrl}`, {
+        method: 'POST',
+        headers: headers,
+        body: `trackedUrl=${url}`,
+      })
+        .then((response) => {
+          console.log(response.status)
```

Comment:
- [ ] we may log on info message including the body but we shouldn't do console.log --> looks like an error!